egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -63,6 +63,8 @@ def GetArgs():
                         "The noise distribution will be mixed with a uniform distribution according to the smoothing weight")
     parser.add_argument("--max-noises-per-minute", type=int, default = 2,
```

Comment:
This looks good. But perhaps the name and the comments can be changed to reflect that these are foreground noises. e.g. --max-fg-noises-per-minute and so on.

Reply:
This will make all recipe scripts incompatible but ok, I'll work on that.

Reply:
Ok. Perhaps you can add the new option for future use and temporarily keep --max-noises-per-minute with a message that it is deprecated. But use the old option if that is supplied. But @danpovey might have other opinions on that.